pikaday.js

```diff
@@ -470,18 +468,17 @@
         if (opts.splitTimeView) {
             addClass(self.el, 'pika-split-time');
 
-            date = new Date();
+            date = new Date(selDate);
             results = '<select class="pika-select pika-select-time" size="9">';
             for (var h = 0; h < 24; h++) {
                 for (var m = 0; m < 60; m += minutesStep) {
                     date.setHours(h, m, 0, 0);
 
-                    isJustOnTime = date.valueOf() === self._minTime.valueOf();
-                    if (isJustOnTime) {
-                        disabled = false;
-                    } else {
-                        disabled = (isMinDate && date <= self._minTime) || (isMaxDate && date >= self._maxTime);
-                    }
+                    disabled = false;
+
+                    if (date.getTime() < self._minTime.getTime()) disabled = true;
```

Comment:
It's not about readability as much as it's about:

https://stackoverflow.com/a/42416541/2465738